src/multicloud-connector/HISTORY.rst

```diff
@@ -3,6 +3,14 @@
 Release History
 ===============
 
+1.1.0
+++++++
+* GA release
```

Comment:
1.0.0b1 -> GA version:  1.0.0
Please remove `azext.isPreview: true` tag in  azext_metadata.json file as well. 
For more detail, please refer to doc here: https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md

Reply:
Updated the release version to 1.0.0